modules/mobile/spec/request/maintenance_windows_request_spec.rb

```diff
@@ -42,16 +44,16 @@
               'type' => 'maintenance_window',
               'attributes' => {
                 'service' => 'claims',
-                'startTime' => '2021-05-25T21:33:39.000Z',
-                'endTime' => '2021-05-26T01:45:00.000Z'
+                'startTime' => '2021-05-25T23:33:39.000Z',
+                'endTime' => '2021-05-27T01:45:00.000Z'
```

Comment:
Notice claims end time being extended due to new vbms window that ends later. Figured it didn't require it's own spec.